### PR TITLE
net/http: delete blank lines

### PR DESCRIPTION
**Pull request by GoldenSheep402**

### CODE REVIEW

File: src/net/http/server.go [hunk 1]
```diff
@@ -3616,7 +3616,6 @@ func numLeadingCRorLF(v []byte) (n int) {
 		break
 	}
 	return
-
```
Comment (alexandear) [inline src/net/http/server.go:3619]: Thanks.

Could you fix all blank lines in `net/http` package?

E.g., the following ones:

https://github.com/golang/go/blob/0cd309e12818f988693bf8e4d9f1453331dcf9f2/src/net/http/client_test.go#L861-L863
https://github.com/golang/go/blob/0cd309e12818f988693bf8e4d9f1453331dcf9f2/src/net/http/client_test.go#L1137-L1139
https://github.com/golang/go/blob/0cd309e12818f988693bf8e4d9f1453331dcf9f2/src/net/http/clientserver_test.go#L1619-L1621
https://github.com/golang/go/blob/0cd309e12818f988693bf8e4d9f1453331dcf9f2/src/net/http/h2_bundle.go#L7234-L7236
https://github.com/golang/go/blob/0cd309e12818f988693bf8e4d9f1453331dcf9f2/src/net/http/h2_bundle.go#L10238-L10240
https://github.com/golang/go/blob/0cd309e12818f988693bf8e4d9f1453331dcf9f2/src/net/http/transfer.go#L1053
https://github.com/golang/go/blob/0cd309e12818f988693bf8e4d9f1453331dcf9f2/src/net/http/transport_test.go#L387-L389
https://github.com/golang/go/blob/0cd309e12818f988693bf8e4d9f1453331dcf9f2/src/net/http/transport_test.go#L1043-L1045
https://github.com/golang/go/blob/0cd309e12818f988693bf8e4d9f1453331dcf9f2/src/net/http/transport_test.go#L4778-L4780
https://github.com/golang/go/blob/0cd309e12818f988693bf8e4d9f1453331dcf9f2/src/net/http/cookiejar/jar_test.go#L1087-L1089
https://github.com/golang/go/blob/0cd309e12818f988693bf8e4d9f1453331dcf9f2/src/net/http/httptrace/trace_test.go#L87-L89
https://github.com/golang/go/blob/0cd309e12818f988693bf8e4d9f1453331dcf9f2/src/net/http/httputil/reverseproxy_test.go#L152-L154
https://github.com/golang/go/blob/0cd309e12818f988693bf8e4d9f1453331dcf9f2/src/net/http/httputil/reverseproxy_test.go#L1521-L1523
https://github.com/golang/go/blob/0cd309e12818f988693bf8e4d9f1453331dcf9f2/src/net/http/internal/chunked_test.go#L212-L214

**Review by alexandear — CHANGES_REQUESTED** [review]

File: src/net/http/server.go (hunk 1 above)
Comment (GoldenSheep402) [reply src/net/http/server.go:3619]: ok

Comment (GoldenSheep402) [reply src/net/http/server.go:3619]: finished

Comment (alexandear) [reply src/net/http/server.go:3619]: Maybe we should also update a PR's title?

```
net/http: delete blank lines
```

**Review by alexandear — APPROVED** [review]

File: src/net/http/server.go (hunk 1 above)
Comment (GoldenSheep402) [reply src/net/http/server.go:3619]: oh,tks. I forgot that

**Review by maxwelbm — APPROVED** [review]

**Pull request closed without merging**